Skip tests that fail for various reasons.

Fails in chroot:
  + acceptance/01_vars/02_functions/network/008.cf
  + acceptance/10_files/templating/timed/expired_edit_line_locks.cf
  + unit/buffer_test
  + unit/enterprise_extension_test
  + load/run_last_seen_threaded_load

Fails on sparc:
  + unit/mon_cpu_test

Fails during reproducibility testing on amd64:
  + unit/rlist_test


--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -21,4 +21,4 @@
 # (COSL) may apply to this file if you as a licensee so wish it. See
 # included file COSL.txt.
 #
-SUBDIRS = unit load acceptance
+SUBDIRS = unit
--- a/tests/acceptance/01_vars/02_functions/network/008.cf
+++ /dev/null
@@ -1,66 +0,0 @@
-#######################################################
-#
-# Test host2ip()
-#
-#######################################################
-
-body common control
-{
-      inputs => { "../../../default.cf.sub" };
-      bundlesequence  => { default("$(this.promise_filename)") };
-      version => "1.0";
-}
-
-#######################################################
-
-bundle agent init
-{
-  vars:
-      "dummy" string => "dummy";
-}
-
-#######################################################
-
-bundle agent test
-{
-
-  vars:
-      # Neither of these are likely to change...
-      "localhost" string => host2ip("localhost");
-      "a" string => host2ip("a.root-servers.net");
-}
-
-#######################################################
-
-bundle agent check
-{
-  vars:
-      "localhost" string => "127.0.0.1";
-      "localhost_6" string => "::1";
-      "a" string => "198.41.0.4";
-      "a_6" string => "2001:503:ba3e::2:30";
-
-  classes:
-      "ok_a" or => {
-                     strcmp("$(test.a)", "$(a)"),
-                     strcmp("$(test.a)", "$(a_6)"),
-      };
-      "ok_localhost" or => {
-			     strcmp("$(test.localhost)", "$(localhost)"),
-			     strcmp("$(test.localhost)", "$(localhost_6)"),
-      };
-      "ok" and => {
-                    "ok_a",
-                    "ok_localhost",
-      };
-
-  reports:
-    DEBUG::
-      "Expected $(test.localhost) == $(localhost)";
-      "Expected $(test.a) == ( $(a) or $(a_6) )";
-    ok::
-      "$(this.promise_filename) Pass";
-    !ok::
-      "$(this.promise_filename) FAIL";
-}
-
--- a/tests/acceptance/10_files/templating/timed/expired_edit_line_locks.cf
+++ /dev/null
@@ -1,44 +0,0 @@
-# Check whether file promises have a shorter expiry time than edit_line promises
-# resulting from templates. This will create an empty file because the file is
-# opened for editing, but all the edit_line promises have expired.
-
-body common control
-{
-    inputs => { "../../../dcs.cf.sub", "../../../plucked.cf.sub" };
-    bundlesequence => { default($(this.promise_filename)) };
-}
-
-bundle agent test
-{
-        
-  commands:
-      # Note, no -K, we are testing locks.
-      "$(sys.cf_agent) -v -D AUTO,DEBUG -f $(this.promise_filename).sub"
-        contain => in_shell;
-}
-
-bundle agent check
-{
-  methods:
-    test_pass_1::
-      "any" usebundle => dcs_wait($(this.promise_filename), 70);
-
-  vars:
-    test_pass_2::
-      "content_edit_line" string => readfile("$(G.testfile).edit_line", 10000);
-      "content_cftemplate" string => readfile("$(G.testfile).cftemplate", 10000);
-      "content_mustache" string => readfile("$(G.testfile).mustache", 10000);
-
-  classes:
-    test_pass_2::
-      "ok_edit_line" expression => strcmp($(content_edit_line), "text"),
-        scope => "namespace";
-      "ok_cftemplate" expression => strcmp($(content_cftemplate), "text"),
-        scope => "namespace";
-      "ok_mustache" expression => strcmp($(content_mustache), "text"),
-        scope => "namespace";
-
-  methods:
-    test_pass_2::
-      "any" usebundle => dcs_passif("ok_edit_line.ok_cftemplate.ok_mustache", $(this.promise_filename));
-}
--- a/tests/unit/Makefile.am
+++ b/tests/unit/Makefile.am
@@ -99,7 +99,6 @@
 	atexit_test \
 	csv_writer_test \
 	item_test \
-	rlist_test \
 	domainname_test \
 	set_test \
 	set_domainname_test \
@@ -142,7 +141,6 @@
 	files_interfaces_test \
 	refcount_test \
 	list_test \
-	buffer_test \
 	connection_management_test \
 	expand_test \
 	string_expressions_test \
@@ -158,7 +156,6 @@
 	rb-tree-test \
 	variable_test \
 	protocol_test \
-	mon_cpu_test \
 	mon_load_test \
 	mon_processes_test \
 	mustache_test \
@@ -219,11 +216,8 @@
 conversion_test_SOURCES = conversion_test.c ../../libpromises/conversion.c
 
 if !BUILTIN_EXTENSIONS
-    check_PROGRAMS += enterprise_extension_test
-
     enterprise_extension_test_SOURCES = enterprise_extension_test.c
 
-    check_LTLIBRARIES += cfengine-enterprise.la
     cfengine_enterprise_la_SOURCES = enterprise_extension_test_lib.c
     cfengine_enterprise_la_LDFLAGS = $(AM_LDFLAGS) \
 	-avoid-version -module -shared -export-dynamic -rpath /
